[TUTORIAL] Add Coinjoin Coordinator Creation

kravens opened the pull request:
Let me know your feedback on this draft of the coinjoin coordinator tutorial, I think we can simplify significantly if we leave out the clearnet coordinator and add Tor service only coordinator.

kravens committed:
Draft coordinator tutorial

First draft code generated for coinjoin coordinator tutorial

kravens committed:
Added links to Wasabi Wallet repo

kravens committed:
Added example deployment code

kravens committed:
Taproot warning

kravens committed:
nostr tool

BitPreto committed:
remove @ from github username

BitPreto committed:
put a space between headers and the text below

kravens committed:
Updated tutorial

Using the executable deployment instead of building from source code.
Also focus on Tor-only, skipping over clearnet domain configuration.

kravens commented:
It will get you all the way to running the coordinator, but the onion address should not stay a mystery.

kravens committed:
Tor and magic link sharing update

Clarified how to find hidden service address and how to share a magic link with coordinator URI and parameters.

kravens committed:
Update en.md

kravens commented:
I've added a section on inviting other users, where you can depend on the Nostr announcer or have a shareable magic link that you can give to new users of your coordinator (anonymously or under pseudonym if desired).

kravens committed:
Renamed images

PaoloFoti commented:
may you just change your branch name from dev to anything else? or you open a new PR from another branch...cause i cant work on it via lazygit, due to the same branch name from Plan ₿ Network's dev branch

kravens committed:
Re save state for the overall task

kravens committed:
Corrected image file locations

Cover and image 01 now named correctly

kravens committed:
Merge branch 'dev' of https://github.com/kravens/bitcoin-educational-content into dev

kravens committed:
Repairing Co-pilot fuckup

You can ignore this file change in merge to PlanB-Network repo.

PaoloFoti commented:
Hello @kravens , thanks for your edits. Yes, the image path is now good. You should just remove the README file, and add a `tutorial.yml` empty file in the same directory of the `en.md` file. in this way I can fill it properly and the PR will be ready to be merged (if you won't have to edit it more)

PaoloFoti commented:
you can take [this file](https://github.com/kravens/bitcoin-educational-content/blob/revert-commit-ef7f471/tutorials/privacy/coinjoin-coordinator/tutorial.yml) and put it here, i'll edit and we will be ready. otherwise you update that branch, we close this PR and open a new one from that branch. you tell me :)

kravens committed:
Delete README.md

kravens commented:
Should I make tutorial.yml empty? It is already filled in with my metadata now. Deleting the README file created a conflict...

PaoloFoti commented:
no need to do nothing more with the `tutorial.yml` file! I can see it now, I'll edit and mark it as ready soon

regarding the conflict form the README.md file i'd say dont worry, we would have to delete such anyway. my colleague merging this PR will likely handle that

BitPreto committed:
Update en.md

BitPreto reviewed (changes requested):
Hello @kravens there is only one thing left, could you please have the image 01 more horinzontal-shaped? now it's 1000×860 → could it be at least 1400×860? The rest is amazing

kravens committed:
Update 01.webp

BitPreto committed:
Update tutorial.yml

BitPreto reviewed (approved):
just waiting for a confirmation on our side regarding the license/licence filed and then the PR will be put ready for merge. Ty @kravens for your quick update

kravens committed:
Update en.md

Refined the Tor configuration section to use the automatic setup by wcoordinator, this is the easiest setup possible.

kravens committed:
Merge branch 'dev' of https://github.com/kravens/bitcoin-educational-content into dev

kravens commented:
I've updated the Tor config section to use the easiest automatically configured method implemented in the Wasabi Coordinator.

kravens committed:
Added human-monitor image

There is an API accessible by web to check the current round, I've added this with an example image of Tor Browser.

BitPreto committed:
Update en.md

BitPreto committed:
Update tutorial.yml

BitPreto reviewed (approved):
@LoicPandul ready for merge

LoicPandul committed:
Merge remote-tracking branch 'upstream/dev' into pr/3690

LoicPandul committed:
Update cover.webp

LoicPandul committed:
Update en.md

LoicPandul reviewed (approved):
Perfect, thank you very much for this very interesting tutorial @kravens!
